plugins/modules/zabbix_globalmacro.py

```diff
@@ -0,0 +1,222 @@
+
+DOCUMENTATION = r'''
+---
+module: zabbix_globalmacro
+short_description: Create/update/delete Zabbix Global macros
+version_added: 1.4.0
+description:
+   - manages Zabbix Global macros, it can create, update or delete them.
+   - For macro_type 1 (Secret Text) the value field cannot be validated and will always be overwritten due to the secret nature of the Text.
+author:
+    - "Cove (@cove)"
+    - Dean Hailin Song (!UNKNOWN)
+    - Timothy Test (@ttestscripting)
+requirements:
+    - "python >= 2.6"
+    - "zabbix-api >= 0.5.4"
+options:
+    macro_name:
+        description:
+            - Name of the global macro in zabbix native format C({$MACRO}) or simple format C(MACRO).
+        required: true
+        type: str
+    macro_value:
+        description:
+            - Value of the global macro.
+            - Required if I(state=present).
+        type: str
+    macro_type:
+        description:
+            - Type of the global macro Text or Secret Text.
+            - Required if I(state=present).
+            - 0 Text
+            - 1 Secret Text
+        type: str
+        choices: ['0', '1']
+        default: "0"
+    macro_description:
+        description:
+            - Text Description of the global macro.
+        type: str
+    state:
+        description:
+            - State of the macro.
+            - On C(present), it will create if macro does not exist or update the macro if the associated data is different.
+            - On C(absent) will remove a macro if it exists.
+        required: false
+        choices: ['present', 'absent']
+        type: str
+        default: "present"
+    force:
+        description:
+            - Only updates an existing macro if set to C(yes).
+        default: 'yes'
+        type: bool
+extends_documentation_fragment:
+- community.zabbix.zabbix
+'''
+
+EXAMPLES = r'''
+- name: Create new global macro or update an existing macro's value
+  local_action:
+    module: community.zabbix.zabbix_globalmacro
+    server_url: http://monitor.example.com
+    login_user: username
+    login_password: password
+    macro_name: EXAMPLE.MACRO
+    macro_value: Example value
+    macro_type: 0
+    macro_description: Example description
+    state: present
+# Values with curly brackets need to be quoted otherwise they will be interpreted as a dictionary
+- name: Create new global macro in Zabbix native format with Secret Type
+  local_action:
+    module: community.zabbix.zabbix_globalmacro
+    server_url: http://monitor.example.com
+    login_user: username
+    login_password: password
+    macro_name: "{$EXAMPLE.MACRO}"
+    macro_value: Example value
+    macro_type: 1
+    macro_description: Example description
+    state: present
+- name: Delete existing global macro
+  local_action:
+    module: community.zabbix.zabbix_globalmacro
+    server_url: http://monitor.example.com
+    login_user: username
+    login_password: password
+    macro_name: "{$EXAMPLE.MACRO}"
+    state: absent
+'''
+
+
```

Comment:
```suggestion
RETURN=r"""
"""
```
Is there any information that can be included in the RETURN block?
If any can be included in the return value, I think it is useful information for the module users.
What do you think?